client/rest/rest.go

```diff
@@ -0,0 +1,26 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+// addHTTPDeprecationHeaders is a mux middleware function for adding HTTP
+// Deprecation headers to a http handler
+func addHTTPDeprecationHeaders(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Deprecation", "true")
+		w.Header().Set("Link", "<https://docs.cosmos.network/v0.40/interfaces/rest.html>; rel=\"deprecation\"")
```

Comment:
```suggestion
		w.Header().Set("Link", "<https://docs.cosmos.network/v0.40/interfaces/rest.html>; rel=\"deprecation\"")
		w.Header().Set("Warning", "199 - \"this endpoint is deprecated and may not work with pre-v0.40 transactions, see deprecation link for more info\"")
```

Reply:
why `pre-`? We should just say "may not work as before"

Reply:
soryr, meant post-v0.40 transtions. Updated with your suggestion.